### PR TITLE
feat(bs): add page-header and nav-tabs style

### PR DESCRIPTION
**Pull request by AxelPeter**

### CODE REVIEW

File: src/utils/_settings.less [hunk 1]
```diff
@@ -1,5 +1,10 @@
+// Dependencies path
 @bs-path: "../node_modules/bootstrap";
 @oui-path: "../node_modules/ovh-ui-kit";
 
+// Fonts folder
 @oui-icon-dist-folder: "./icons";
 @oui-font-source-sans-pro-folder: "./fonts/source-sans-pro";
+
+// OVH Universe
+@ovh-universe-color: @brand-primary;
```
Comment (antleblanc) [inline src/utils/_settings.less]: Does adding this variable into the `README` file might be useful?

File: src/bootstrap/navs.less [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                &:hover, &:focus {
+                    > * {
+                        color: #fff;
+                    }
+                }
+            }
+        }
+
+        &.active {
+            &:after {
```
Comment (antleblanc) [inline src/bootstrap/navs.less]: `&:after` => `&::after`

File: src/utils/_settings.less (hunk 1 above)
Comment (AxelPeter) [reply src/utils/_settings.less]: Yes, it can be. I'll add it

File: src/bootstrap/navs.less (hunk 2 above)
Comment (AxelPeter) [reply src/bootstrap/navs.less]: done

**Pull request merged**